export const fallbackMessage = `Unexpected error occurred. Please try again later.`
